README.md

```diff
@@ -179,8 +185,13 @@ Add the following github secrets for the CD pipeline to work automatically:
 - MONGO_CONNECTION_STRING = Your Mongo Connection String
 - DBNAME = Mongo DB Name
 - COLLECTIONAME = Mongo Collection Name
+- CLIENT_ID = Github OAuth Client Id
+- CLIENT_SECRET = Github OAuth Client Secret
+- OAUTHLIB_INSECURE_TRANSPORT = 1
```

Comment:
Agreed, the tests don't go through the login process, but these are required to be setup as github secrets and Heroku vars, isn't it?

Reply:
If the tests don't need these secrets, what is the point of adding them as github secrets?

Yes, Heroku needs these values, because Heroku actually hosts your application and you want to log into it. But this section of your README says "Add the following github secrets for the CD pipeline to work".

Reply:
Makes sense, will remove them. Thanks for clarifying